src/backend/base/langflow/api/utils.py

```diff
@@ -296,3 +297,28 @@ def custom_params(
     if page is None and size is None:
         return None
     return Params(page=page or MIN_PAGE_SIZE, size=size or MAX_PAGE_SIZE)
+
+
+def create_output_for_toolset_component(component_node: dict):
+    component_node["tool_mode"] = True
+    component_node["outputs"] = [
+        Output(
+            types=["Tool"],
+            selected="Tool",
+            name="component_as_tool",
+            hidden=None,
+            display_name="Toolset",
+            method="to_toolkit",
+            value=UNDEFINED,
+            cache=True,
+            required_inputs=None,
+        )
+    ]
+    return component_node
```

Comment:
This function does the same as `_build_tool_output` with more chances of being outdated in case we change the constants.